2DIterativeRecon.jl

```diff
@@ -0,0 +1,431 @@
+## Matlab Author: Rodrigo de Barros Vimieiro
+## Translated by Sonia Minseo Kim
+
+## 2-D Branchless Distance Driven Code
+using Plots: plot
+using ImagePhantoms: shepp_logan, SheppLoganToft
+using ImageGeoms: ImageGeom, axesf
+using MIRTjim: jim, prompt #specify the functions you are using in the packages
+using LazyGrids: ndgrid 
+using Dierckx: Spline1D
+
+# Julia is compiling program so you need to restart the program in order to clear the variables 
+
+## Geometry Definitions
+# Geometry
+deg = 1
+geo = (DSD = 100000, DSO = 99700, pSize = 1, dSize = 0.5, nPix = 256, nDet = 1024
+        , theta = deg2rad.(0:deg:360-deg), isoX = 0, isoY = 0) #Define tuple
+
+# Make the Phantom Image
+phantomImg = shepp_logan(geo.nPix, SheppLoganToft())
```

Comment:
this test data should go at *bottom* of file (and eventually in a separate test file)